src/vsr.zig

```diff
@@ -1085,33 +1096,94 @@ pub const Headers = struct {
     /// - DVC headers (disjoint chain)
     pub const ViewChangeSlice = ViewChangeHeadersSlice;
     pub const ViewChangeArray = ViewChangeHeadersArray;
+
+    fn dvc_blank(op: u64) Header {
+        return std.mem.zeroInit(Header, .{
+            .command = .prepare,
+            .op = op,
+            .checksum = 0,
```

Comment:
Even if a header does legitimately have `checksum=0|1`, some field would be different, so the `std.meta.eql`s in `dvc_header_type` wouldn't match.

Reply:
But maybe set `.command` to something other than `.prepare`, such that, if we accidently use a fake header, we'd blow up? Eg, I think if we pass these blaks by accident to `replace_header` stuff will "just work" rather than "just bork"?